assignments/objects.js

```diff
@@ -17,48 +19,157 @@ const example = {
   "gender": "F"
 }
 
+// defining a class with a constructor so I can assembly-line these.
+class Intern {
+  constructor(id,email,name,gender) {
+    this.id = id;
+    this.email = email;
+    this.name = name;
+    this.gender = gender;
+  }
+}
+console.log(Intern);
+
 // Write your intern objects here:
 
+// Today I aspire to transcendent laziness.
+let internArray = ['1,mmelloy0@psu.edu,Mitzi,F', '2,kdiben1@tinypic.com,Kennan,M', '3,kmummery2@wikimedia.org,Keven,M', '4,gmartinson3@illinois.edu,Gannie,M', '5,adaine5@samsung.com,Antonietta,F']
```

Comment:
you handles this question way differently from most people  lol. But I love it, great programmer mindset!